scripts/sct_register_to_template.py

```diff
@@ -575,8 +576,10 @@ def main(args=None):
     sct.printv('\nGenerate output files...', verbose)
     sct.generate_output_file(path_tmp + 'warp_template2anat.nii.gz', path_output + 'warp_template2anat.nii.gz', verbose)
     sct.generate_output_file(path_tmp + 'warp_anat2template.nii.gz', path_output + 'warp_anat2template.nii.gz', verbose)
-    sct.generate_output_file(path_tmp + 'template2anat.nii.gz', path_output + 'template2anat' + ext_data, verbose)
-    sct.generate_output_file(path_tmp + 'anat2template.nii.gz', path_output + 'anat2template' + ext_data, verbose)
+    fname_template2anat = os.path.join(path_output, 'template2anat' + ext_data)
+    fname_anat2template = os.path.join(path_output, 'anat2template' + ext_data)
+    sct.generate_output_file(path_tmp + 'template2anat.nii.gz', fname_template2anat, verbose)
+    sct.generate_output_file(path_tmp + 'anat2template.nii.gz', fname_anat2template, verbose)
```

Comment:
This will have to be updated after merging of #1535 (`os.path.join(path_tmp, ...)`).